backend/my_app/api/v1/views/auth_views.py

```diff
@@ -0,0 +1,81 @@
+"""
+This module creates authentication views.
+Both registration and login views.
+"""
+from flask import jsonify, make_response, request
+from flask_restful import Resource
+from flask_jwt_extended import create_access_token
+from ..models.user_model import User
+from ..validators import UserPostSchema, UserLoginSchema
+
+
+class Register(Resource, User):
+    """ This class creates a view for registering a new user"""
+
+    def __init__(self):
```

Comment:
Let's remove this `__init__` method since it's unnecessary. You are not creating an instance of `Register` anywhere. Do this for all view files.